oak_containers/proto/interfaces.proto

```diff
@@ -58,11 +58,19 @@ service Launcher {
   // This API is called exactly once after the Attestation Evidence is generated. Calling this API
   // a second time will result in an error.
   rpc SendAttestationEvidence(SendAttestationEvidenceRequest) returns (google.protobuf.Empty) {}
+
+  // Notifies the launcher that the trusted app is ready to serve requests and listening on the
+  // specified port.
+  rpc NotifyAppReady(google.protobuf.Empty) returns (google.protobuf.Empty) {}
 }
 
 // Defines the service exposed by the orchestrator, that can be invoked by the application.
 service Orchestrator {
   // Exposes the previously loaded trusted application config to the application,
   // which may choose to retrieve it.
   rpc GetApplicationConfig(google.protobuf.Empty) returns (GetApplicationConfigResponse) {}
+
+  // Notifies the orchestrator that the trusted app is ready to serve requests and listening on the
+  // specified port.
+  rpc NotifyAppReady(google.protobuf.Empty) returns (google.protobuf.Empty) {}
```

Comment:
Great, thanks! 

Can we add in the comment where the port is specified? I think it is pre-determined at startup, no?

Reply:
Thanks, I used to pass it in the notification, but removed it without updating the comment. I will clarify in a follow-up PR.